pkgs/top-level/python-packages.nix

```
@@ -5328,6 +5328,8 @@ in {
 
   uarray = callPackage ../development/python-modules/uarray { };
 
+  ueberzug = callPackage ../tools/graphics/ueberzug { };
```

if the package is good about dependency management, then I would move the expression into python-modules, then kept everything else is the same.

Otherwise:
```suggestion
  ueberzug = toPythonModule( callPackage ../tools/graphics/ueberzug { } );
```